hooks/conan-center.py

```diff
@@ -59,6 +59,7 @@
              "KB-H058": "ILLEGAL CHARACTERS",
              "KB-H059": "CLASS NAME",
              "KB-H060": "NO CRLF",
+             "KB-H061": "TOOLS CROSS BUILDING"
```

Comment:
```suggestion
             "KB-H062": "TOOLS CROSS BUILDING"
```

Because 61 is under the PR: https://github.com/conan-io/hooks/pull/320/files